### DISCUSSION TITLE
Molstar vs. NGL: MD Simulation

### DISCUSSION THREAD

**Discussion by frederickparsons-tandemai** [molstar/molstar#508] [General]
Hello community, I'm currently investigating Molstar vs. NGL. First, can anyone point me to a nice piecewise comparison between the two? Namely, are there things NGL can do that Mol* can't yet. I can't seem to find anything of that sort. Second, how does Mol* fair in comparison to NGL with respect to displaying MD simulation data? Thank you! I look forward to exploring Mol* more in details.

**Comment by arose** [comment]
I am not aware of piecewise comparison. There is not much in NGL that is not in Mol*, some less used file formats like mmtf are not in Mol*. Displaying MD simulation data is well supported in Mol*, some formats as in NGL.

Generally, NGL has a more approachable but less powerful/customizable API than Mol*. Mol* has much better/nicer graphics and allows you to build your own library/viewer, adjusting almost everything but you will need to read the code for that.

**Reply by frederickparsons-tandemai** [reply]
Thanks Alexander! Can you comment on manipulating the coordinates of molecular systems in NGL vs. Mol*?

For example, I'd like to be able to manipulate the coordinates of a ligand's atoms. Would you recommend NGL or Mol* if I want to explore that down the road? Maybe even changing the topology of loaded structures eventually.

**Reply by arose** [reply]
Both NGL and Mol* are primarily viewers and don't have built in support for editing. However it is usually fast enough to edit the structure externally and just reload it. If you just want to change coordinates you can re-use the trajectory handling machinery.
